internal/experiments/experiments.go

```diff
@@ -0,0 +1,35 @@
+package experiments
+
+import "fmt"
+
+type ExperimentConfig struct {
```

Comment:
This seems unnecessary. I would just add a field to the config for this specific feature.